### PR TITLE
Dev  forecast with effect

### PR DESCRIPTION
**Pull request by samioan**
-Added getForecastSucceededEpic
-Minor improvements regarding intro bg image
-Forecast screen improved

### CODE REVIEW

File: src/components/game/components/intro/components/forecast/Forecast.js [hunk 1]
```diff
@@ -1,77 +1,35 @@
...
+const Forecast = ({
+  name,
+  temp,
+  description,
+  onClickForecastHandler,
+  showMessage,
+}) => {
+  useEffect(() => {
+    onClickForecastHandler();
+  }, []);
```
Comment (stacon) [inline src/components/game/components/intro/components/forecast/Forecast.js]: What is its purpose? I don't see a click event or something. If its just a lifecycle method it, delegate it to a HOC

File: src/models/forecast/epics.js [hunk 2]
```diff
@@ -1,48 +1,29 @@
...
-        .catch((err) => {
-          console.error(err);
-        });
+    effect(getForecastService, getForecast)
+  );
 
+// TODO: getForecastSucceededEpic
+const getForecastSucceededEpic = (action$) =>
+  action$.pipe(
+    ofType("//FORECAST/GET_FORECAST_SUCCEEDED"),
```
Comment (stacon) [inline src/models/forecast/epics.js]: Use the appropriate action creator and not hardcoded string

**Review by stacon — CHANGES_REQUESTED** [review]
Check comments and act accordingly! Ask me if you need something!

File: src/models/forecast/epics.js (hunk 2 above)
Comment (samioan) [reply src/models/forecast/epics.js]: Done.

File: src/components/game/components/intro/components/forecast/Forecast.js (hunk 1 above)
Comment (samioan) [reply src/components/game/components/intro/components/forecast/Forecast.js]: Done.

**Comment by samioan** [comment]
Minor fixes regarding the forecast and log epics.
Background image now present in both intro and game page.
withModelProps added to withForecastProps.
Added css fade into effect in intro screen and a few more messages regarding temperature.

SOS: Replace action no longer works, needs fixing!

**Review by stacon — APPROVED** [review]

**Pull request merged**